packages/create-resolve-app/README.md

```diff
@@ -1,86 +1,99 @@
-
-
 # **Create reSolve App** 🚀
 [![npm version](https://badge.fury.io/js/create-resolve-app.svg)](https://badge.fury.io/js/create-resolve-app)
 
-This package creates an empty application based on the reSolve framework ([`hello-world` example](https://github.com/reimagined/resolve/tree/master/examples/hello-world)). Use the `--example` option to create a single page application (SPA) based on another example. The application is built on the CQRS and Event Sourcing principles using [React](https://github.com/facebook/react) + [Redux](https://github.com/reactjs/redux) on the client.
+This package creates an empty application based on the reSolve framework ([`hello-world` example](https://github.com/reimagined/resolve/tree/master/examples/hello-world)). Use the `--example` option to create another sample application. reSolve applications are built on the CQRS and Event Sourcing principles and use [React](https://github.com/facebook/react) + [Redux](https://github.com/reactjs/redux) on the client side.
 
-Create reSolve App allows you to specify application blocks (aggregates, read models, and UI part React components present) in a semi-declarative manner. With the `resolve-scripts` package, you do not need to write an API backend manually. Instead, `resolve-scripts` deploys backend and domain services to interact with the client which is wrapped in the `resolve-redux` package for an automated interaction.
+**Create reSolve App** allows you to specify application blocks (aggregates, read models, and UI elements (React components)) in a semi-declarative manner. The `resolve-scripts` package deploys API backend and domain services to interact with the client. The `resolve-redux` package automates the client-server interaction.
```

Comment:
I'm not sure about the `resolve-redux` package, please check this info

Reply:
> Which command accepts the --config argument?

`npm run dev --config=my-custom-resolve.config.json`
`npm run build --config=my-custom-resolve.config.json`

Reply:
> Which command accepts the --config argument?

npm run dev --config=my-custom-resolve.config.json
npm run build --config=my-custom-resolve.config.json

Reply:
Text okay